src/js/_enqueues/admin/site-icon.js

```diff
@@ -99,40 +130,98 @@
+			.each( function( i, img ) {
+				$( img )
+				.attr({
+					'src': attributes.url,
+					'alt': i18nAltString 
+				});
+			});
+			$iconPreview.removeClass( 'hidden' );
 
-	function switchToUpdate( url ){
-		// Set site-icon-img src to the url and remove the hidden class.
-		$( '#site-icon-preview').find('img').not('.browser-preview').each( function(i, img ){
-			$(img).attr('src', url );
-		});
-		$( '#site-icon-preview' ).removeClass( 'hidden' );
-		// Remove hidden class from remove.
+		// Remove hidden class from the remove button.
 		$( '#js-remove-site-icon' ).removeClass( 'hidden' );
-		// If the button is not in the update state, swap the classes.
-		if( $( '#choose-from-library-link' ).attr( 'data-state' ) !== '1' ){
-			var classes = $( '#choose-from-library-link' ).attr( 'class' );
-			$( '#choose-from-library-link' ).attr( 'class', $( '#choose-from-library-link' ).attr('data-alt-classes') );
-			$( '#choose-from-library-link' ).attr( 'data-alt-classes', classes );
-			$( '#choose-from-library-link' ).attr( 'data-state', '1' );
+
+		// If the choose button is not in the update state, swap the classes.
+		if ( $chooseButton.attr( 'data-state' ) !== '1' ) {
+			$chooseButton.attr({
+				'class': $chooseButton.attr( 'data-alt-classes' ),
+				'data-alt-classes': $chooseButton.attr( 'class' ),
+				'data-state': '1'
+			});
 		}
 
-		// swap the text of the button
-		$( '#choose-from-library-link' ).text( $( '#choose-from-library-link' ).attr( 'data-update-text' ) );
+		// Swap the text of the choose button.
+		$chooseButton.text(
+			$chooseButton.attr( 'data-update-text' )
+		);
 	}
 
-	$( '#js-remove-site-icon' ).on( 'click', function() {
-		$( '#site_icon_hidden_field' ).val( 'false' );
-		$( '#site-icon-preview' ).toggleClass( 'hidden' );
-		$( this ).toggleClass( 'hidden' );
-
-		var classes = $( '#choose-from-library-link' ).attr( 'class' );
-		$( '#choose-from-library-link' ).attr( 'class', $( '#choose-from-library-link' ).attr( 'data-alt-classes' ) );
-		$( '#choose-from-library-link' ).attr( 'data-alt-classes', classes );
-
-		// Swap the text of the button.
-		$( '#choose-from-library-link' ).text( $( '#choose-from-library-link' ).attr( 'data-choose-text' ) );
-		// Set the state of the button so it can be changed on new icon.
-		$( '#choose-from-library-link' ).attr( 'data-state', '');
-	});
-}(jQuery));
+	/**
+	 * Initializes the functionality to remove the site icon.
+	 *
+	 * @since 6.5.0
+	 */
+	( function () {
+		var $chooseButton = $( '#choose-from-library-button' ),
+			$iconPreview = $( '#site-icon-preview' );
+
+		$( '#js-remove-site-icon' ).on( 'click', function() {
+			$( '#site_icon_hidden_field' ).val( 'false' );
+			$( this ).toggleClass( 'hidden' );
+			$iconPreview.toggleClass( 'hidden' );
+			$iconPreview.find( 'img' ).not( '.browser-preview' )
+			.each( function( i, img ) {
+				$( img )
+				.attr({
+					'src': '',
+					'alt': '' 
+				});
+			});
```

Comment:
I think these lines should be indented by one tab.

Reply:
And maybe indent .attr-lines as well one more line? What do you think @afercia ?

Like this:
```
$iconPreview.find( 'img' ).not( '.browser-preview' )
	.each( function( i, img ) {
		$( img )
			.attr({
				'src': '',
				'alt': '' 
			});
	});
```

Reply:
Oh yes, I missed that 👍🏽